test/prune-publish.test.js

```diff
@@ -72,6 +72,7 @@ test('publish many messages that might need pruning', (t) => {
         new Promise((res, rej) => {
           ssb.tribes2.publish(
             { type: 'potato', content: value, recps: [group.id] },
+            null,
```

Comment:
in another PR lets see if we can make `opts` actually optional.
I only do this when it's an API other devs are gonna be using